carbon-design-system/carbon-addons-iot-react
# fix(timeseriescard): tooltip was not formatting in the correct locale

Closes #

**Summary**

- bug found where the timeseriescard tooltip was not being formatted correctly in different languages.  Note: the x-axis was being formatted correctly for different timeseriescards

**Change List (commits, features, bugs, etc)**

- fix(handleTooltip): initialize dayjs module with locale before using
- fix(*Card): pass the locale along

**Acceptance Test (how to verify the PR)**

- go to this story and switch locales to French in the knob in the deploy preview.  ?path=/story/1-watson-iot-timeseriescard--multi-line-interval-data-choices. Mouse over a liine of the chart and view the tooltip and verify the month comes first

**Regression Test (how to make sure this PR doesn't break old functionality)**

- Verify english locales format correctly



**Things to look for during review**
